pkg/gateway/operations/base.go

```diff
@@ -39,7 +39,7 @@ const (
 	OperationIDOperationNotFound    OperationID = "not_found"
 )
 
-type ActionIncr func(action, repository, ref, userID string)
+type ActionIncr func(action, userID, repository, ref string)
```

Comment:
Did you change all the usages? I missed it

Reply:
This is a fix - all the calls use this order of arguments.